packages/web/stores/derived-data/concentrated-liquidity/historical-and-liquidity-data.ts

```diff
@@ -119,6 +119,16 @@ export class ObservableHistoricalAndLiquidityData {
     );
   }
 
+  @action
+  setLastChartData = (setPrice: number) => {
+    this.historicalChartData[this.historicalChartData.length - 1].high =
+      setPrice;
+    this.historicalChartData[this.historicalChartData.length - 1].low =
+      setPrice;
+    this.historicalChartData[this.historicalChartData.length - 1].close =
+      setPrice;
+  };
```

Comment:
Eh this may not be a good way to do it, modifying the query store..

Instead, I'd get the current price from pool and return it in `lastChartData`:
```
  @computed
  get lastChartData(): TokenPairHistoricalPrice | null {
    const price = Number(
      this.queries.queryPools.getPool(this.poolId)
        ?.concentratedLiquidityPoolInfo?.currentPrice || 0
    );
    return {
      close: price,
      high: price,
      low: price,
      open: price,
      time: new Date().getTime(),
    };
  }
```

Reply:
But lastChartData isn't what's used when rendering the price chart